modules/ROOT/pages/asm-secret-group-creation-task.adoc

```diff
@@ -166,14 +166,12 @@ Add a list of certificates that were revoked by certificate authorities before t
 
 == Edit a Secret Group
 
-When you edit a secret group, other users are locked out while you edit the necessary security objects. If a user doesn't end an editing session by either finishing or canceling it, the *Edit* button is disabled for other users. If you click *Cancel Edit*, the open editing session is closed, and the *Edit* button becomes available, even if the session was started by a different user. 
+Edit a secret group to add secret types such as Keystore, Truststore, and Certificates.
```

Comment:
can we change from "to add" to "to manage", also after Certificates, please add " etc"?
Edit a secret group to manage secret types such as Keystore, Truststore, Certificates and etc.

Reply:
Changed.